Docker/Dockerfile.dev.deploy

```diff
@@ -0,0 +1,17 @@
+FROM dalgo_main_image:0.1 as build
```

Comment:
Can you add descriptions at the top of each Dockerfile so it's easy to understand the use case for each of them?